packages/scenes/src/variables/variants/query/QueryVariable.tsx

```diff
@@ -100,6 +102,12 @@ export class QueryVariable extends MultiValueVariable<QueryVariableState> {
     );
   }
 
+  public cancel() {
+    this.setState({ loading: false });
+    const sceneVarSet = getClosest(this, (s) => s instanceof SceneVariableSet ? s : undefined);
```

Comment:
```suggestion
    const sceneVarSet = this.parent as SceneVariableSet;
```

We assume variables are inside variable sets (at least for now) :)